codebuild/bin/install_awslc.sh

```diff
@@ -16,31 +16,27 @@ set -eu
 pushd "$(pwd)"
 
 usage() {
-    echo "install_awslc.sh build_dir install_dir is_fips"
+    echo -e "\ninstall_awslc.sh build_dir install_dir"
+    echo -e "\tIf you need FIPS, use the FIPS specific install script.\n"
     exit 1
 }
 
-if [ "$#" -ne "3" ]; then
+if [ "$#" -ne "2" ]; then
     usage
 fi
 
 BUILD_DIR=$1
 INSTALL_DIR=$2
-IS_FIPS=$3
 
 if [[ ! -f "$(which clang)" ]]; then
   echo "Could not find clang"
   exit 1
 fi
 
-# These tags represents the latest versions that S2N is compatible
-# with. It prevents our build system from breaking when AWS-LC
-# is updated, last done on 2023-02-22.
-if [ "$IS_FIPS" == "1" ]; then
-  AWSLC_VERSION=AWS-LC-FIPS-1.0.3
-else
-  AWSLC_VERSION=v1.36.0
-fi
+# Ask GitHub for the latest release.
+AWSLC_VERSION=$(curl --silent "https://api.github.com/repos/aws/aws-lc/releases/latest" | \
+	grep -Po '"tag_name": "\K.*?(?=")')
```

Comment:
Looking at the aws-lc releases, it seems like we may get fips releases this way. For example: https://github.com/aws/aws-lc/releases/tag/AWS-LC-FIPS-3.0.0

I wonder if we should pin after all to avoid this, unless we did something more complex with the releases API, like not using `/latest` and filtering for `v1*` on the tag name or something.

Reply:
Looking at the FIPS commit data: https://api.github.com/repos/aws/aws-lc/releases/tags/AWS-LC-FIPS-3.0.0
You could probably check the "target_commitish" field. The non-FIPS commits will always be "main".

Reply:
It seems like target_commitish isn't main for some release commits for some reason: https://api.github.com/repos/aws/aws-lc/releases/tags/v1.48.1

Reply:
Yeah, I added a regex match to the non-fips versionin the latest revision 👍